### DISCUSSION TITLE
Running Godot from Vscode?

### DISCUSSION THREAD

**Discussion by gerroon** [godotengine/godot-vscode-plugin#618] [Q&A] [answered]
Hi

Is there a way to run my project from Vscode with this extension? It would be nice if I can just press a button and it would send start scene command to currently running Godot to test the Gdscript code, or something along those lines.

thanks

**Comment by atirutw** [comment, marked as answer]
Press F5 and find the option for Godot. It's equivalent to pressing the play button in the editor

**Reply by gerroon** [reply]
Thanks!
